src/Forms/BatchConvert.cs

```diff
@@ -2468,7 +2471,7 @@ private void ScanFiles(IEnumerable<string> fileNames)
                                 var format = sub.LoadSubtitle(fileName, out _, enc, true, null, false);
                                 if (format == null)
                                 {
-                                    foreach (var f in SubtitleFormat.GetBinaryFormats(true))

```

Comment:
for every file dropped a list of new formats will be created...  (don't do that!)